core/src/main/java/edu/wpi/grip/core/settings/ProjectSettings.java

```diff
@@ -98,6 +101,14 @@ public void setDeployJavaHome(String deployJavaHome) {
         if (deployJavaHome != null) this.deployJavaHome = deployJavaHome;
     }
 
+    public String getDeployJvmOptions() {
+        return deployJvmOptions;
+    }
+
+    public void setDeployJvmOptions(String deployJvmOptions) {
+        this.deployJvmOptions = deployJvmOptions;
```

Comment:
This needs to check that the `deployJvmOptions` aren't null.
See the above examples.